packages/unmock/src/index.ts

```diff
@@ -0,0 +1,80 @@
+import FailingBackend from "./backend/failing-backend";
```

Comment:
should this package be renamed to `unmock-core`, a-la https://github.com/apollographql/apollo-server/tree/master/packages/apollo-server-core?

Reply:
it should - good catch!

Reply:
this means that there will no longer be a useable `unmock` package on `npm` but that's of course fine for now, we can make it an alias for `unmock-node` later.

Reply:
We could add an `unmock` package that simply includes all these sub-packages or some default setup.